### PR TITLE
2021-03-25-simonamarcu2-start-with-html

### PR DESCRIPTION
**Pull request by simonamarcu2**

### CODE REVIEW

**Review by RadosavAntonio — APPROVED** [review]
good

File: web/simonamarcu2/html/primul/simy.css [hunk 1]
```diff
@@ -1,43 +1,68 @@
...
   text-align: center;
-  font-family: cursive;
-}
+} */
 
 p {
-  text-align: center;
-  font-family: sans-serif;
+  text-align: left;
+  text-decoration: lavender;
```
Comment (tandaraelena) [inline web/simonamarcu2/html/primul/simy.css]: Maybe you wanted to say: `text-decoration-color: lavender;` ?

**Review by tandaraelena — CHANGES_REQUESTED** [review]

**Review by psyhedeliq — APPROVED** [review]
🚀 🚀 🚀

**Review by RadosavAntonio — DISMISSED** [review]
there are some tabs (the div starting on 23) and some empty lines that should not be there. The website is looking well but is just for good practice.

File: web/simonamarcu2/html/primul/simy.css (hunk 1 above)
Comment (simonamarcu2) [reply web/simonamarcu2/html/primul/simy.css]: Thank you for observation!

**Review by biggicode — APPROVED** [review]

**Review by tandaraelena — APPROVED** [review]

**Review by robertoalx — APPROVED** [review]
Here it goes 💯

**Review by RalucaRbc — APPROVED** [review]
Well done Simona 💯

**Pull request merged**